awx/settings/defaults.py

```diff
@@ -1001,7 +1001,7 @@ def IS_TESTING(argv=None):
             'level': 'INFO',  # very verbose debug-level logs
         },
         'awx.analytics': {
-            'handlers': ['external_logger'],
+            'handlers': ['task_system', 'external_logger'],
```

Comment:
This absolutely hammers the log file `/var/log/tower/task_system.log` with messages like:

```
2020-09-17 01:20:52,346 INFO     awx.analytics.performance api response times
2020-09-17 01:20:52,683 INFO     awx.analytics.job_events Event data saved.
2020-09-17 01:20:59,555 INFO     awx.analytics.activity_stream Activity Stream update entry for job_template
```

These have no more information than this, and it's not usable. This was never intended to be sent to file based logs, only to the external logger. What motivated this specific change?

A simple test run now generates 1.5 MB for this whereas tower.log is 130 KB, and I would expect them to be pretty similar.

Reply:
The process of sending analytics is split between awx.main.analytics and awx.main.tasks.

Before this change, both of these would log, and they'd end up in entirely separate places. To find errors, you'd have to look in a different place based on which particular line of code threw the error.

Reply:
Whoops, this adjusted the wrong logger.  PR incoming.